Source/UI/About/IdentityView.swift

```diff
@@ -123,7 +123,7 @@ struct IdentityView: View {
                     }
                     .zIndex(extendedHeader ? 500 : 1000)
                     .offset(y: headerOffset)
-                MessageListView(strategy: strategy)
+                MessageListView(messages: [], strategy: NoHopFeedAlgorithm(identity: identity))
```

Comment:
Why don't we want to use the `OneHopFeedAlgorithm` here for pubs anymore?

Reply:
👌